hr_timesheet_sheet/views/hr_timesheet_sheet_views.xml

```diff
@@ -210,21 +281,22 @@
         groups="hr_timesheet.group_timesheet_manager"
     />
 
-    <record id="act_hr_timesheet_sheet_form" model="ir.actions.act_window">
-        <field name="name">Timesheet Sheets to Approve</field>
+    <record id="act_hr_timesheet_sheet_to_review" model="ir.actions.act_window">
```

Comment:
The same thing to prevent

Reply:
No known modules found in OCA, but I got your point, thanks for feedback